changelog/unreleased/features/6943.md

```diff
@@ -0,0 +1,5 @@
+- Added `RoadObjectType.IC` and `RoadObjectType.JCT` for interchanges and junctions and the corresponding classes `Interchange` and `Junction`.
+- Limited number of reported alternatives to 2.
+- Sped up onboard routing cancellation.
+- Fixed null pointer dereference.
```

Comment:
I'm not the SDK user, but I don't understand what it means. Do we need it in the changelog?
```suggestion
```

Reply:
Well, I think it was causing a native crash, so why not mention it.

Reply:
This gives no context as to which crash and what was causing it. I agree that we should either provide more info (in this case repro steps) or skip it.